spec.html

```diff
@@ -42032,7 +42032,7 @@ <h1>Atomics.sub ( _typedArray_, _index_, _value_ )</h1>
 
     <emu-clause id="sec-atomics.wait">
       <h1>Atomics.wait ( _typedArray_, _index_, _value_, _timeout_ )</h1>
-      <p>`Atomics.wait` puts the calling agent in a wait queue and puts it to sleep until it is notified or the sleep times out. The following steps are taken:</p>
+      <p>`Atomics.wait` puts the calling agent in a wait queue and suspends it until it is notified or the wait times out, returning a String differentiating those cases. The following steps are taken:</p>
```

Comment:
While not grammatically wrong, I don't think, I'd prefer "... and suspends it until notified or until the wait times out", i.e. duplicating the "until", which I think reads better.